build.gradle

```diff
@@ -38,7 +38,7 @@ dependencies {
 	// libraries
 	implementation ('commons-io:commons-io:2.6')
 	implementation ('org.zeroturnaround:zt-zip:1.13')
-	implementation ('com.google.code.gson:gson:2.8.5')
+	implementation ('com.google.code.gson:gson:2.8.6')
```

Comment:
Ive had this being empty before, no idea what casues it, deleting the file seems to fix it.

Doing this should be fine, but I wouldnt be surprised if it breaks agian.

Reply:
Ok well I still had gson 2.8.6 cached so I updated it to that version. It didn't break anything so far